libsolidity/analysis/DocStringAnalyser.cpp

```diff
@@ -139,9 +182,55 @@ void DocStringAnalyser::handleCallable(
 	StructurallyDocumentedAnnotation& _annotation
 )
 {
-	static set<string> const validTags = set<string>{"author", "dev", "notice", "return", "param"};
+	static set<string> const validTags = set<string>{"author", "dev", "notice", "return", "param", "inheritdoc"};
 	parseDocStrings(_node, _annotation, validTags, "functions");
 	checkParameters(_callable, _node, _annotation);
+
+	if (CallableDeclaration const* baseFunction = resolveInheritDoc(_callable, _node, _annotation))
+		copyMissingTags(_annotation, baseFunction->annotation());
+	else if (
+		_annotation.docTags.empty() &&
+		_callable.annotation().baseFunctions.size() == 1 &&
+		parameterNamesEqual(_callable, **_callable.annotation().baseFunctions.begin())
```

Comment:
When comparing `_callable` and `baseFunctions`, which function should we really check? I think when we use `begin()`, the choice is almost arbitrary. Shouldn't we check the first 'parent' that implements `_callable`?

Reply:
Note that I check the size of the baseFunctions for `== 1`, which means there is only one to check and no choice to make. If there are more we don't auto-inherit ([as specified in the ticket](https://github.com/ethereum/solidity/issues/8911#issuecomment-631496750)).